docs/dev/CodingGuidelines/CodingGuidelines.md

```diff
@@ -177,6 +178,15 @@ The local functions can be used in the following three cases:
 
 The [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions) should be considered in addition to our own.
 
+### Diagnostic and Code Fix Messages Style
+
+Diagnostic and code fix messages should follow the following rules:
```

Comment:
@EAndrosova  please verify the addition to coding guidelines

Reply:
approved